packages/sw-background-sync-queue/demo/index.html

```diff
@@ -123,9 +123,8 @@
         const toastContainer = document.querySelector('.toast-container');
 
         function purchase(){
-            fetch('https://jsonplaceholder.typicode.com/posts/1')
-            .then(res=>res.json())
-            .then(data=>{
+            fetch('http://localhost:3000/packages/sw-build/')
```

Comment:
why are you fetch sw-build? What exactly does this need to achieve in the request?

If it just needs some data, could it use one of the test server end points like "/__echo/counter"